### DISCUSSION TITLE
Add the report button to comment or post in Discussions

### DISCUSSION THREAD

**Discussion by ghost** [community/community#16749] [Discussions]
Hi all!

#### 1. Feature name
Add the report button to comment or post in Discussions

#### 2. Feature description
If I or anyone at any given time deems a comment or thread post to be bad, I or anyone can ask someone from the open source community team to moderate the comment or post

#### 2.1 Functional requirements
For this idea to be accepted, a report button is required

#### 2.2 Concept
Who moderates the content or the discussions is the open source community(users)

#### 2.3.1 Idea
![image](https://user-images.githubusercontent.com/95204402/168713742-ffd190a7-86b0-4627-b259-ad352803226b.png)

#### 2.3.2 Image description
As we can see in the image, there is a report button - with this report button, users can request discussion moderation and comments

#### 3. Notes
1. I'd like to help GitHub and the open source community.
2. Sites like YouTube or Discord, social networks have a comment or content moderation system.
3. I don't want to promote any social networks(Youtube or Discord), I'm thinking about how to improve GitHub. I quote Youtube and Discord only for bibliographic reference
4. Maybe it would be interesting to have the possibility of this feature in the GitHub Discussion to have a better quality in the question of the comment to be posted

#### 4. References
1. https://www.discourse.org/
2. https://support.google.com/youtube/answer/10888907?hl=en

**Comment by StingyJack** [comment]
I would like to be able to report low quality discussions like [this one](https://github.com/orgs/github-community/discussions/18861) and have them either removed or improved.

**Reply by ghost** [reply]
@StingyJack thank you for feedback ;D

**Comment by ghost** [comment]
@StingyJack Hi! How are you? What do you think about this idea?

#### 1. Concept
I wonder if there isn't a way to integrate [Akismet](https://akismet.com/) into GitHub Discussions to avoid spam, malware, etc. 

#### 2. What is Akismet? 
Used by millions of websites, Akismet filters out hundreds of millions of spam comments from the Web every day. Add Akismet to your site so you don't have to worry about spam again.

#### 3. Why?
1. "Akismet stops spam"
2. Wordpress, Discourse use Akismet to avoid spam.
3. It would be interesting if Github Discussions had an integration with Akismet to prevent spam.

#### 4. Notes
In addition to the report button, it could have an integration with Akismet.

**Reply by StingyJack** [reply]
I dont think I have an opinion on Akismet specifically as I'm not seeing a problem with spam (yet), just with incomplete or low quality posts.

**Reply by ghost** [reply]
@StingyJack thank you for feeback ;D
